entry/src/main/ets/MainAbility/pages/ChatHeadsSample.ets

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2022 Application Library Engineering Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ChatHead, ChatheadModel } from "@ohos/chatheads"
+
+@Entry
+@Component
+struct ChatHeadsSample {
+  @State chatheadModel: ChatheadModel = new ChatheadModel();
+
+  aboutToAppear() {
+    this.chatheadModel = this.chatheadModel
```

Comment:
need not assign back to this.chatheadModel??

Reply:
it calls the attributes to set position, add chathead images and set constants(k,b)

Reply:
@neeraj-patidar The changes done on the ChatModel instance within the library will reflect automatically, as it is call by reference. So its not required to re-assign the model back to itself after doing any changes on the object. (The reason for returning object from the library callbacks is to make sure we are able to do a chaining calling mechanism from the sample app.)

Reply:
ohh! got it. doing changes.